internal/api/pprof_stub.go

```diff
@@ -0,0 +1,32 @@
+// This file contains the code to enable pprof profiling. It is only
+// included in the build when the 'pprof' build tag is set in the Dockerfile.
```

Comment:
I think this needed to be updated or remove, this file seem to be used when `pprof` is not set.

Reply:
Whoops, sorry, missed this comment. You are correct, I've updated the comment. Thanks.